src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/RoundUp.txt

```diff
@@ -230,3 +230,8 @@ Error({Kind:ErrorKind.InvalidArgument})
 >> RoundUp("true","false")
 Error(Table({Kind:ErrorKind.InvalidArgument},{Kind:ErrorKind.InvalidArgument}))
 
+>> ForAll(Sequence(16,-1,-1), RoundUp( Float(123.12345678919), Value ))
```

Comment:
>Float(123.12345678919) [](http://example.com/codeflow?start=39&length=22)

Instead of using the Float function (which is not available in Power Apps), can you use something like `Sqrt(123.45 * 213.45)`, which will produce a Float value?